test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -1476,16 +1483,27 @@ public void queryClusterStatus() throws Exception {
         assertThat(nodeStatusMap.get(SERVERS.ENDPOINT_2)).isEqualTo(NodeStatus.UP);
         assertThat(clusterStatus.getClusterStatus()).isEqualTo(ClusterStatus.STABLE);
 
+        // Write 10 entries. 0-9.
```

Comment:
Add a tests case where all nodes are down because this PR also addresses the issue where this method hangs when all nodes are down.